R/iSEE.R

```diff
@@ -616,6 +619,8 @@ iSEE <- function(
   # Launching the app.
   #######################################################################
 
-  shinyApp(ui = iSEE_ui, server = iSEE_server)
+  app <- shinyApp(ui = iSEE_ui, server = iSEE_server)
```

Comment:
I suggest returning the app directly, as this is what `app.R` in server mode would expect. Or at least, allowing a `run=FALSE` option.

Reply:
iSEE what you mean. Coming soon in a push near you.